bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4535,4 +4535,8 @@
             values ('bahmni.admin.csv.upload.dateFormat', 'yyyy-M-d', 'Default date format for all CSV imports', uuid());
         </sql>
     </changeSet>
+    <changeSet id="bahmni-core-RenameColumn-202201021608" author="Rohit, Deepthi, Soorya, Kavitha">
```

Comment:
should have a precondition check to validate the "precise" field exists and only then apply the changeset. Also why use a SQL file, just put inline

Reply:
Hi @angshu, Added the precondition to validate precise